libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/NameInfo.kt

```diff
@@ -0,0 +1,11 @@
+package com.mapbox.navigation.base.trip.model
+
+/**
+ *
+ * @param name
+ * @param shielded
+ */
+data class NameInfo(
+    val name: String,
+    val shielded: Boolean
+)
```

Comment:
Same as above.

Reply:
@mskurydin @SiarheiFedartsou could you help here? Are we envisioning extending this / adding more data as part of `NameInfo` in the future?

Reply:
90% will not extend.